src/components/dashboard/MagicLinkInfo.js

```diff
@@ -91,7 +91,6 @@ const getStylesFromProps = ({ theme }) => {
     illustration: {
       flexGrow: 1,
       flexShrink: 0,
-      maxWidth: '100%',
       maxHeight: getDesignRelativeHeight(230),
```

Comment:
what about using false here also?
attach images on how it looks on large screens after using false

Reply:
without isMax param
![image](https://user-images.githubusercontent.com/8187382/71968372-765d2180-320d-11ea-81fd-4415bf3a3827.png)
with isMax param
![image](https://user-images.githubusercontent.com/8187382/71968479-b02e2800-320d-11ea-85b9-146ed8ed2bc2.png)

Reply:
@LiavGut @sirpy 
which result is acceptable?

Reply:
@StanislavShevchenko without isMax param (the first one)

Reply:
@LiavGut Ok, done